operating/manage-user-access/understanding-authentication-providers.adoc

```diff
@@ -0,0 +1,93 @@
+:_content-type: ASSEMBLY
+[id="understanding-authentication-providers"]
+= Understanding authentication providers
+include::modules/common-attributes.adoc[]
+:context: authentication-provider-structure
+
+toc::[]
+
+[role="_abstract"]
+
+An authentication provider connects to a third-party source of user identity, for example, an identity provider or IDP, gets the user identity, issues a token based on that identity, and returns the token to RHACS. 
```

Comment:
Generally we don't use parenthesis but i think you need to put (for example, an identity provider or IDP) in parenthesis because you have three verb actions after that modifier to "third party-source of user identity". Or else rewrite the sentence so in this case, I think parentheses are preferable.